server/src/main/java/io/deephaven/server/console/python/PythonImportInitializer.java

```diff
@@ -0,0 +1,17 @@
+package io.deephaven.server.console.python;
+
+import io.deephaven.engine.util.PyCallableWrapperJpyImpl;
+import io.deephaven.integrations.python.PythonObjectWrapper;
+
+public abstract class PythonImportInitializer {
```

Comment:
Feels like this pattern could be replaced with service loader.

Reply:
I think the problem w/ service loader is that the classes are on the classpath regardless of whether we are running python or groovy. In this current pattern, it's only invoked when we are coming from known-python script session.